proposals/js/JsName.md

```diff
@@ -28,6 +28,8 @@ Use **binary retention** since it should be available from deserialized descript
 + `FUNCTION` (constructors are not included)
 
 + `PROPERTY`
+<br/>
+New name provided by the annotation should be used for generating name of extension properties too which generated as functions.
```

Comment:
Currently, we generate `get/set_$propertyName_$mangleHash` identifier. What is the proposed pattern for accessor functions when there's `@JsName` on a property declaration?